gemrb/plugins/KEYImporter/KEYImporter.h

```diff
@@ -39,7 +39,7 @@ class ResourceDesc;
 struct BIFEntry {
 	std::string name;
 	ieWord BIFLocator;
-	char path[_MAX_PATH];
+	std::string path;
```

Comment:
Why not path_t?

Reply:
we can, it just requires VFS.h

Reply:
I think that would be preferable in case we ever need to change the underlying type.